ext/imap/php_imap.stub.php

```diff
@@ -2,150 +2,107 @@
 
 /** @generate-function-entries */
 
```

Comment:
Per the comment in the .c file, let's define the `IMAPConnection` class in here.

Reply:
Done :)